src/inspirehep-holdingpen-js/holdingpen/holdingpen.controllers.js

```diff
@@ -78,6 +83,23 @@
     function setDecision(id, decision) {
       HoldingPenRecordService.setBatchDecision($scope.vm.invenioSearchResults.hits.hits, [+id], decision);
     }
+
+    function onBestMatchSelected(workflowId){
+      HoldingPenRecordService.setMatchDecision(workflowId, Number($scope.selectedBestMatchesById[workflowId]))
+        .then(function () {
+          $scope.matchedById[workflowId] = true;
+          $scope.matchDecisionMadeById[workflowId] = true;
+        });
+    }
+
+    function onNoMatchSelected(workflowId) {
+      HoldingPenRecordService.setMatchDecision(workflowId, null)
+      .then(function () {
+          $scope.matchedById[workflowId] = false;
+          $scope.matchDecisionMadeById[workflowId] = true;
```

Comment:
shouldn't be ``false``?

Reply:
nope, this is when they decide none of them matches.